feature_mining/em_base.py

```diff
@@ -0,0 +1,69 @@
+class ExpectationMaximization:
```

Comment:
It will be really nice to have this set up in the inheritance structure set up for this, thank you!